mteb/models/colbert_models.py

```diff
@@ -50,7 +51,14 @@ def encode(
         Returns:
             The encoded sentences.
         """
-        return np.asarray(self.static_model.encode(sentences, **kwargs))
+        if "task_name" in kwargs:
+            kwargs.pop("task_name")
+        if "prompt_type" in kwargs:
+            kwargs.pop("prompt_type")
```

Comment:
I think it's better to left them in kwargs, because pylate support these parameters https://github.com/lightonai/pylate/blob/8de1842f1dac3c80108843b1676cffe500b1a793/pylate/models/colbert.py#L385-L386

Reply:
Nice, but it will require renaming as they have different names:
```python
prompt_name: str | None = None,
prompt: str | None = None,
```
Perhaps that's where the performance difference lies.

Reply:
In this part should be like all sentence transformers https://github.com/embeddings-benchmark/mteb/blob/2550a27664a875f03d6fcf36f035c8cc93e07100/mteb/models/sentence_transformer_wrapper.py#L87-L91